devtools/buildIndexFiles.js

```diff
@@ -13,11 +13,15 @@ const isComponentFile = (source) => {
     return !ignoredFiles.some(ignored => source.includes(ignored));
 };
 
+const isComponentExported = (source) => {
+    return !source.match(/^\_/);
+};
+
```

Comment:
Internal components are named with a leading underscore (`_`) so we should skip over those when building the index files.